tests/e2e/actions.go

```diff
@@ -1025,12 +1024,12 @@ func (tr TestRun) redelegateTokens(action redelegateTokensAction, verbose bool)
 		`--keyring-backend`, `test`,
 		`-y`,
 	)
-
 	if verbose {
 		fmt.Println("redelegate cmd:", cmd.String())
 	}
 
 	bz, err := cmd.CombinedOutput()
+	time.Sleep(3 * time.Second)
```

Comment:
What are these sleeps for?

Reply:
I think they're actually unnecessary, they were what we used to get the tests working and this branch should instead be a plain merge of main. So I think I'm going to close this PR.